doc/instances.md

```diff
@@ -57,6 +57,16 @@ How to import instances:
 :diataxis:Migrate from LXC </howto/migrate_from_lxc>
 ```
 
+```{only} diataxis
+How to run a nested Docker workload in an instance:
+```
+
+```{filtered-toctree}
+:titlesonly:
+
+:diataxis:Docker GPU workload in a LXD container </howto/container_gpu_passthrough_with_docker>
```

Comment:
The title should be an action (and also, the current one is too long and doesn't fit into one line).
Maybe "Run docker GPU workloads" if that fits?

Reply:
See also my comment on the topic title.

Reply:
fixed